recipe/meta.yaml

```diff
@@ -1,37 +1,43 @@
-{% set version = "1.2.0" %}
+{% set name = "sphinx-issues" %}
+{% set version = "5.0.0" %}
 
 package:
-  name: sphinx-issues
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: sphinx-issues-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/s/sphinx-issues/sphinx-issues-{{ version }}.tar.gz
-  md5: 0f40881c9c95c40c6e752abce61882cc
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name.replace("-", "_") }}/{{ name.replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 192e43cf071ed7aead401cd14fd15076ecb0866238c095d672180618740c6bae
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
+    - flit-core <4
     - pip
-    - python
-    - setuptools
+    - python >=3.9
   run:
-    - python
+    - python >=3.9
     - sphinx
 
 test:
   imports:
     - sphinx_issues
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://github.com/sloria/sphinx-issues
+  summary: "A Sphinx extension for linking to your project's issue tracker"
   license: MIT
+  license_family: MIT
   license_file: LICENSE
-  summary: "A Sphinx extension for linking to your project's issue tracker"
+  dev_url: https://github.com/sloria/sphinx-issues
 
 extra:
   recipe-maintainers:
```

Comment:
@m-rossi We might want to try and get the admins to to add additional maintainers to this feedstock. I offered to help maintain it in early 2022 and @bjlittle offered earlier this summer but there's been no word back.

Reply:
Yes, I saw that issue when I started the PR. Let's wait 1-2 days for an answer and then add you & me as recipe maintainers and ping conda-forge-core.